src/jit-analyze/jit-analyze.cs

```diff
@@ -943,21 +943,29 @@ void DisplayFileMetric(string headerText, int metricCount, IEnumerable<FileDelta
                                         }).ToList();
             var sortedMethodImprovements = methodDeltaList
                                             .Where(x => x.deltaMetric.Value< 0)
-                                            .OrderBy(d => d.deltaMetric.Value).ToList();
```

Comment:
Could you please do it for files as well?

https://github.com/dotnet/jitutils/pull/333/files#diff-990416f0b5c67dde466cbbd414b65c9a82cedafe1192f775e31c8d747c0f412dR900-R907

Reply:
sure. done